doc/FORTANIX-SGX-ABI.md

```diff
@@ -2,6 +2,21 @@
 
 This document describes the ABI of SGX enclaves built using `libenclave`.
 
+## Enclave runner compatibility
+
+a. rust-lang toolchain version: https://github.com/rust-lang/rust/blob/master/src/libstd/sys/sgx/abi/entry.S
+```
+2:  .long 0                    /* desc - toolchain version number, 32-bit LE */
+```
```

Comment:
Instead of pointing at a source file, can you describe this in terms of ELF sections?

Reply:
done